requirements.txt

```diff
@@ -1,6 +1,6 @@
 # Python dependencies required for development
 astunparse
-expecttest
+expecttest==0.1.6
```

Comment:
Maybe let's just ignore `0.2.0`

```suggestion
expecttest!=0.2.0
```

That will also install `0.1.6` for now, but will automatically pull a fixed version when ezyang/expecttest#14 is merged and released.

Reply:
Done